lib/middleware/request_tracker.rb

```diff
@@ -282,12 +282,12 @@ def rate_limit(request)
         type = 60
         limiter60.performed!
         false
-      rescue RateLimiter::LimitExceeded
+      rescue RateLimiter::LimitExceeded => e
         if warn
           Discourse.warn("Global IP rate limit exceeded for #{ip}: #{type} second rate limit", uri: request.env["REQUEST_URI"])
-          !(GlobalSetting.max_reqs_per_ip_mode == "warn")
+          GlobalSetting.max_reqs_per_ip_mode != "warn" && e.available_in
```

Comment:
Ditto ^^. What does `available_in` mean ?

Reply:
Applied the same change as before. `available_in` is the name of the field from the `RateLimiter::LimitExceeded` exception which holds the number of seconds until the request is no longer blocked.